### PR TITLE
removed unnecessary "active" attribute from <a> tag on line 73

### LINKED ISSUE #5272: Remove the by default selected "Asked" button in the Question's tab on the profile's page
Hi, this is a first-timers-only issue. This means we've worked to make it more legible to folks who either haven't contributed to our codebase before, or even folks who haven't contributed to open source before.

If that's you, we're interested in helping you take the first step and can answer questions and help you out as you do. Note that we're especially interested in contributions from people from groups underrepresented in free and open source software!

We know that the process of creating a pull request is the biggest barrier for new contributors. This issue is for you 💝

If you have contributed before, consider leaving this one for someone new, and looking through our general help wanted issues. Thanks!

🤔 What you will need to know.
Nothing. This issue is meant to welcome you to Open Source :) We are happy to walk you through the process.

📋 Step by Step
  🙋 Claim this issue: Comment below. If someone else has claimed it, ask if they've opened a pull request already and if they're stuck -- maybe you can help them solve a problem or move it along!

  📝 Update the file app/views/users/profile.html.erb in the plots2 repository. The "Asked" button is permanently selected as it is marked as active by default.

In the line 73, change
`<a href="#asked" class="btn btn-default active" data-toggle="tab">Asked</a>`
to
`<a href="#asked" class="btn btn-default" data-toggle="tab">Asked</a>`

Screen Shot 2019-03-27 at 6 56 49 PM

 💾 Commit your changes

  🔀 Start a Pull Request. There are two ways how you can start a pull request:

If you are familiar with the terminal or would like to learn it, here is a great tutorial on how to send a pull request using the terminal.

You can also edit files directly in your browser and open a pull request from there.

  🏁 Done Ask in comments for a review :)
Please keep us updated
💬⏰ - We encourage contributors to be respectful to the community and provide an update within a week of claiming a first-timers-only issue. We're happy to keep it assigned to you as long as you need if you update us with a request for more time or help, but if we don't see any activity a week after you claim it we may reassign it to give someone else a chance. Thank you in advance!

If this happens to you, don't sweat it! Grab another open issue.

Is someone else already working on this?
🔗- We encourage contributors to link to the original issue in their pull request so all users can easily see if someone's already started on it.

👥- If someone seems stuck, offer them some help! Otherwise, take a look at some other issues you can help with. Thanks!

🤔❓ Questions?
Leave a comment below!

### PR DESCRIPTION
**Pull request by ibnjunaid**
Fixes  #5272 (<=== Add issue number here)

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

* [x] PR is descriptively titled 📑 and links the original issue above 🔗
* [x] tests pass -- look for a green checkbox ✔️ a few minutes after opening your PR -- or run tests locally with `rake test`
* [x] code is in uniquely-named feature branch and has no merge conflicts 📁
* [x] screenshots/GIFs are attached 📎 in case of UI updation
* [x] ask `@publiclab/reviewers` for help, in a comment below

> We're happy to help you get this ready -- don't be afraid to ask for help, and **don't be discouraged** if your tests fail at first!

If tests do fail, click on the red `X` to learn why by reading the logs.

Please be sure you've reviewed our contribution guidelines at https://publiclab.org/contributing-to-public-lab-software 

Thanks!

### CODE REVIEW

File: app/views/users/profile.html.erb [hunk 1]
```diff
@@ -70,7 +70,7 @@
 
     <div class="tab-pane" id="questions">
       <div class="btn-group text-center">
-        <a href="#asked" class="btn btn-default active" data-toggle="tab">Asked</a>
+        <a href="#asked" class="btn btn-default " data-toggle="tab">Asked</a>
```
Comment (divyabaid16) [inline app/views/users/profile.html.erb]: Great work @ibnjunaid !
But, can you please remove an extra space after ```btn-default```
```suggestion
        <a href="#asked" class="btn btn-default" data-toggle="tab">Asked</a>
```

Comment (ibnjunaid) [reply app/views/users/profile.html.erb]: Yes.Of course. Thank you for review.

Comment (ibnjunaid) [reply app/views/users/profile.html.erb]: @divyabaid16 Done .Please review.

**Comment by jywarren** [comment]
Thank you!!!!

**Pull request merged**